Module/ifo/module.ifo.json

```diff
@@ -2471,6 +2524,10 @@
     "type": "resref",
     "value": ""
   },
+  "Mod_OnNuiEvent": {
```

Comment:
These are new events not present in the current stable version of the game. Please undo these as well.